From 680473856c29599c3ea70af42e967704d7fe3b16 Mon Sep 17 00:00:00 2001
From: Mika Laitio <lamikr@gmail.com>
Date: Mon, 28 Oct 2024 01:00:13 -0700
Subject: [PATCH 5/6] build error fix for target already added error

fix build error which can happen when there is more
than one target enabled at the same time.
(gfx1100, gfx1101)

Signed-off-by: Mika Laitio <lamikr@gmail.com>
---
 example/02_gemm_bilinear/CMakeLists.txt | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/example/02_gemm_bilinear/CMakeLists.txt b/example/02_gemm_bilinear/CMakeLists.txt
index aa1061370..0e2f68ae0 100644
--- a/example/02_gemm_bilinear/CMakeLists.txt
+++ b/example/02_gemm_bilinear/CMakeLists.txt
@@ -5,9 +5,7 @@ foreach(gpu IN LISTS GPU_TARGETS)
  if(gpu IN_LIST gpu_list1 AND target EQUAL 0)
     add_example_executable(example_gemm_bilinear_wmma_fp16 gemm_bilinear_wmma_fp16.cpp)
     add_example_executable(example_gemm_bilinear_wmma_int8 gemm_bilinear_wmma_int8.cpp)
-endif()
-if(GPU_TARGETS MATCHES "gfx908" OR GPU_TARGETS MATCHES "gfx90a" OR GPU_TARGETS MATCHES "gfx940")
-   set(target 1)
+    set(target 1)
  endif()
 endforeach()
 
-- 
2.41.1

